prow/config.gen.yaml

```diff
@@ -2235,6 +2235,249 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-unit-maistra
+    trigger: (?m)^/test( | .* )unit,?($|\s.*)
+    decorate: true
+    always_run: true
+    path_alias: istio.io/istio
+    skip_report: false
+    max_concurrency: 2
+    branches:
+      - ^maistra$
+    rerun_command: /test unit
+    spec:
+      containers:
+      - image: "gcr.io/istio-testing/build-tools:master-2022-05-23T15-41-52"
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        command:
+        - entrypoint
+        - make
+        - -e
+        - T=-v
+        - build
+        - racetest
+        - binaries-test
+        env:
+        - name: BUILD_WITH_CONTAINER
+          value: "0"
+        - name: GOFLAGS
+          value: -mod=vendor
+        - name: XDG_CACHE_HOME
+          value: /tmp/cache
+        - name: GOCACHE
+          value: /tmp/cache
+        # FIXME: Uncomment this when we have our 2.2 proxy in place
+        # - name: ISTIO_ENVOY_BASE_URL
+        #   value: https://storage.googleapis.com/maistra-prow-testing/proxy
+        resources:
+          limits:
+            memory: 24Gi
+            cpu: "4"
+          requests:
+            cpu: "4"
+            memory: 4Gi
+
+  - name: istio-lint-maistra
+    trigger: (?m)^/test( | .* )lint,?($|\s.*)
+    decorate: true
+    always_run: true
+    path_alias: istio.io/istio
+    skip_report: false
+    max_concurrency: 2
+    branches:
+      - ^maistra$
+    rerun_command: /test lint
+    spec:
+      containers:
+      - image: "gcr.io/istio-testing/build-tools:master-2022-05-23T15-41-52"
```

Comment:
Can we use our image?

Reply:
we don't have one- and as this branch is a moving target, I think it makes sense to stick with upstream

Reply:
or do you mean use a different image (e.g. 2.2?) for the lint job than the other jobs?